lib/plugins/chest.js

```diff
@@ -12,7 +12,9 @@ function inject (bot, { version }) {
       assert.ok(chestToOpen.type === mcData.blocksByName.chest.id ||
                 chestToOpen.type === mcData.blocksByName.ender_chest.id ||
                 chestToOpen.type === mcData.blocksByName.trapped_chest.id ||
-                (mcData.blocksByName.barrel && chestToOpen.type === mcData.blocksByName.barrel.id))
+                (mcData.blocksByName.barrel && chestToOpen.type === mcData.blocksByName.barrel.id) ||
+                mcData.blocks[chestToOpen.type].name.endsWith('_shulker_box'))
```

Comment:
Probably should be `'shulker_box'` instead of `'_shulker_box'`, as since 1.13 there's a `'shulker_box'` this can be seen in https://minecraft-data.prismarine.js.org/?v=1.13&d=items

I made some checks for this: https://github.com/imharvol/mineflayer/tree/shulker_tests

Reply:
Block IDs are separate from Item IDs as of 1.13. So the `mcData.blocks` will never contain that item at least in 1.13+. Does this occur in 1.11 or 1.12?

Reply:
Update, even if it does exist in 1.12 and 1.11 with the same ID, it'll still be in the item for and not the block form. So `block.type` will still never return value value, even in those versions. So even then, it's safe.

Reply:
The link is a bit misleading, I meant blocks not items. See for example https://minecraft-data.prismarine.js.org/?v=1.13&d=blocks, there are the shulkers with color, but there's one with no color (`shulker_box`) and thus no underscore, so bot.openChest() wouldn't think it's a chest
I think it's easier to see it in the tests. If you run them as it is right now, it fails in versions since 1.13, if you remove the underscore (`shulker_box`) then it passess

Reply:
Ah, okay. I see what you mean. Yeah, the underscore should be removed then.

Reply:
The underscore has been sent to the void.